src/applications/disability-benefits/all-claims/constants.js

```diff
@@ -78,6 +79,8 @@ export const PCIU_STATES = [
   { label: 'Wyoming', value: 'WY' }
 ];
 
+// Not sure what the deal is here, but I'm getting the following from flow for the next two lines:
+// Error: Missing type annotation for `U`.
```

Comment:
Any thoughts? I removed the `U` from `PCIU` as well as all the label and values inside the array. No dice. I even got rid of the `_` just in case something funny was happening there too. Still the same error. It's really quite baffling. :confused:

Reply:
Looks like it's a thing: https://github.com/facebook/flow/issues/6151